tests/test_ERst.cpp

```diff
@@ -122,13 +115,22 @@ BOOST_AUTO_TEST_CASE(TestERst_1) {
     BOOST_CHECK_THROW(std::vector<double> vect4=rst1.getRst<double>("LOGIHEAD",5), std::runtime_error );
     BOOST_CHECK_THROW(std::vector<bool> vect5=rst1.getRst<bool>("ZWEL",5), std::runtime_error );
 
+    // report step number exists, but data is not loaded. Vector should in this case
+    // be loaded on demand. Hence not throwing an exception
+
+    std::vector<int> vect1=rst1.getRst<int>("ICON",10);
+    std::vector<float> vect2=rst1.getRst<float>("PRESSURE",10);
+    std::vector<double> vect3=rst1.getRst<double>("XGRP",10);
+    std::vector<bool> vect4=rst1.getRst<bool>("LOGIHEAD",10);
+    std::vector<std::string> vect5=rst1.getRst<std::string>("ZWEL",10);
+
     rst1.loadReportStepNumber(25);
 
-    std::vector<int> vect1 = rst1.getRst<int>("ICON",25);
-    std::vector<float> vect2 = rst1.getRst<float>("PRESSURE",25);
-    std::vector<double> vect3 = rst1.getRst<double>("XGRP",25);
-    std::vector<bool> vect4 = rst1.getRst<bool>("LOGIHEAD",25);
-    std::vector<std::string> vect5 = rst1.getRst<std::string>("ZWEL",25);
+    vect1 = rst1.getRst<int>("ICON",25);
+    vect2 = rst1.getRst<float>("PRESSURE",25);
+    vect3 = rst1.getRst<double>("XGRP",25);
+    vect4 = rst1.getRst<bool>("LOGIHEAD",25);
+    vect5 = rst1.getRst<std::string>("ZWEL",25);
```

Comment:
We should probably verify that the vectors have the expected content here&mdash;at least some of the elements.

Reply:
agree, contents is now being checked.